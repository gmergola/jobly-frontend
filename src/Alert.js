import React from 'react';

/**Alert: shows errors when they occur in forms */
function Alert({ errors }) {

  return (
    errors.map((err, i) => {
      if (err === "Invalid Credentials") return <div className="alert alert-danger">{err}</div>
      return <div key={i}><b>{err}</b></div>
    })
  );
}

export default Alert;