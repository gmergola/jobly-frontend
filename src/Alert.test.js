import React from "react";
import { render } from "@testing-library/react";
import Alert from "./Alert";


it("renders without crashing", function () {
  render(
    <Alert errors={['ERROR']} />
  );
});

it("matches snapshot", function () {
  const { asFragment } = render(
    <Alert errors={['ERROR']} />
  );
  expect(asFragment()).toMatchSnapshot();
});