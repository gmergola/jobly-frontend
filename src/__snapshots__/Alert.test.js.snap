// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <b />
  </div>
</DocumentFragment>
`;
